### DISCUSSION TITLE
network error when attempting to fetch resource. in Next 9.0.6

### DISCUSSION THREAD

**Discussion by dpkpaa** [vercel/next.js#13787] [Help] [answered]
When navigating form one page to another page first loading error page and after some time loading my actual page. This is happening in fire fox browser only.

**Comment by lfades** [comment, marked as answer]
Please provide a reproduction that the community can take a look at, or otherwise we won't be able to help you.

**Reply by dpkpaa** [reply]
i am using express for custom routing and. i have some country based url where country name and   page slug is attached with dash separated. problem is when i am loading generic page url it's working fine but when i am switching to county based url for the same page i am getting "TypeError: "NetworkError when attempting to fetch resource."" error in  in Firefox browser only
